homeassistant/components/almond/__init__.py

```diff
@@ -291,7 +291,7 @@ async def async_process(
         context: Context,
         conversation_id: str | None = None,
         language: str | None = None,
-    ) -> intent.IntentResponse:
+    ) -> conversation.ConversationResult | None:
         """Process a sentence."""
         response = await self.api.async_converse_text(text, conversation_id)
```

Comment:
do we know if it generates a conversation ID if `None` is passed in ? If so, it should also be added to the response. I guess this is existing code so maybe we just leave it like it was.

Reply:
The [docs](https://wiki.almond.stanford.edu/api-references/web-almond) do say that the Almond response contains a `conversationId` field, so we could pull it out.